tests/__init__.py

```diff
@@ -0,0 +1,17 @@
+import os
+from functools import update_wrapper
+
+
+def temporary_files(*files):
+    def wrap_test(test):
+        def wrapped_test(*args, **kwargs):
+            if any(os.path.exists(file_) for file_ in files):
+                raise IOError
+            try:
+                test(*args, **kwargs)
+            finally:
+                for file_ in files:
+                    os.remove(file_)
```

Comment:
I think we should check if they exists, for if a test failed, the file might not be there.